src/main/java/Parser.java

```diff
@@ -0,0 +1,68 @@
+
+public class Parser {
+    private static TaskList taskManager = new TaskList();
```

Comment:
Parser should be decoupled from tasks and storage. Parsers should just take the user input and return a Command.